OPERATOR.rst

```diff
@@ -173,28 +173,57 @@ label from blocking PR. Last, remove the blocking relationship.
-updating the versions of the docker images for ``gitlab-ce`` and
-``gitlab-runner``. Then run::
-
-    _select dev.gitlab
-    cd terraform/gitlab
-    make apply
-
-It may be necessary to set ``CI_COMMIT_REF_NAME=develop`` to work around
+Operators must check for updates to GitLab on a daily basis. Updates to Gitlab
+can be quickly checked at the ``/help`` endpoint for `GitLab dev`_ and
+`GitLab prod`_. If updates are available, find the new release of the
+`GitLab Docker image`_ and update the `Gitlab Terraform`_ file. When updating
+the GitLab instance, check if there are applicable updates to the
+`GitLab runner image`_. Before starting the update process, check the
+`GitLab release notes`_ for upgrading instructions. When upgrading across
+multiple GitLab versions, follow the prescribed GitLab `upgrade path`_.
+
+Before any changes are applied, run::
+
+	git fetch -all
+	git checkout -b gitlab/<yyy-mm-dd> github/develop
+	_select dev.gitlab
+
+Stop the GitLab EC2 instance (do not terminate it), and create a snapshot of its
+EBS volume. To create a snapshot, run::
+
+	python scripts/create_gitlab_snapshot.py
+
+It may take some time for the snapshot of the volumes to complete, make sure
+the ``Snapshot status`` shows ``completed`` within the AWS Console
+before proceeding with any changes. Edit 
+``terraform/gitlab/gitlab.tf.json.template.py``, updating the versions of
+the docker images for ``gitlab-ce`` and ``gitlab-runner``. Then run::
+
+    CI_COMMIT_REF_NAME=develop make -C terraform/gitlab
+
+It will necessary to change the ``CI_COMMIT_REF_NAME`` to the targeted
+deployment's branch name (e.g. ``develop`` or ``prod``) to work around
 ``check_branch``.
 
-The GitLab instance should be online again in 10 minutes or so. If it takes
-substantially longer, contact the lead.
+The GitLab instance should be online again in 10 minutes. If it takes longer,
+contact the lead. When the GitLab endpoint is online, have the lead confirm
+that any background migrations triggered by the upgrade have finished.
+Background migrations can be found under
+*Admin Area — Monitoring — Background Migrations*.
+
+Once the ``dev`` GitLab instance has been successfully updated, the same
+changes need to be applied to the ``prod`` instance. Once both instances have
+been successfully updated, file a PR with the changes against the ``develop``
+branch and request review from the lead.
+
+.. _GitLab dev: https://gitlab.dev.singlecell.gi.ucsc.edu/help
+.. _GitLab prod: https://gitlab.azul.data.humancellatlas.org/help
+.. _GitLab Docker image: https://hub.docker.com/r/gitlab/gitlab-ce/tags
+.. _GitLab runner image: https://hub.docker.com/r/gitlab/gitlab-runner/tags
+.. _GitLab release notes: https://about.gitlab.com/releases/categories/releases/
+.. _GitLab Terraform: https://github.com/DataBiosphere/azul/blob/develop/terraform/gitlab/gitlab.tf.json.template.py#L1064
+.. _upgrade path: https://docs.gitlab.com/ee/update/index.html#upgrade-paths
```

Comment:
Ideally a link definition should be immediately below the first paragraph that references it.